src/main/java/seedu/trippie/command/NewTripCommand.java

```diff
@@ -98,9 +100,12 @@ public void execute(Ui ui, TrippieData trippieData) {
             do {
                 System.out.print("Enter the foreign currency abbreviation (eg. MYR):");
                 currencyAbbreviation = ui.getLine();
+                if (currencyAbbreviation.length() > 3 | isNumberIncluded(currencyAbbreviation)) {
+                    currencyAbbreviation = null;
+                    System.out.println("Currency abbreviations should be 3-letter alphabetical codes!");
```

Comment:
This will still allow empty or numerical currency abbreviation.

Can use regex to make sure it only contains uppercase alphabets.

Reply:
Added in cases for error handling, thanks!